### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by amrathmd**

### CODE REVIEW

**Comment by amrathmd** [comment]
Screenshot from 2023-03-08 20-56-02

**Comment by amrathmd** [comment]
Screenshot from 2023-03-08 20-59-02

File: src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ const HomePage = () => {
               <Route path='/signup' ><Signup /></Route>
               <Route path='/logout' ><Logout /></Route>
               <Route path='/profile' ><Profile/></Route>
+              <Route path='/dashboard'><Admin /></Route>
```
Comment (maharjunm) [inline src/pages/HomePage/HomePage.tsx:25]: everyone can see this route, we should show this only to the admins.

File: src/pages/AdminPage/Admin.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        console.log(e);
+      }
+    };
+    fetchjobs();
+    
+  }, []);
+  console.log(jobs);
+  return (
+    <div>
+      {jobs.length>0 && jobs.map((p:Job)=>(
```
Comment (maharjunm) [inline src/pages/AdminPage/Admin.tsx]: please use existing components.

File: src/pages/PostJobs/Form.tsx [hunk 3]
```diff
@@ -133,15 +143,27 @@ const Form = () => {
       }, 2000);
     }
   };
-  const onSubmit=async()=>{
-    const res=await axios.post(REACT_BACKEND_URL+'v1/job/',form).then(res=>console.log(res)).catch(e=>console.log(e));
+  const onSubmit=async(event:any)=>{
+    event.preventDefault();
+    try {
+      const res = await fetch(`${REACT_BACKEND_ROUTE}v1/job`, {
```
Comment (maharjunm) [inline src/pages/PostJobs/Form.tsx]: why did you change this to fetch?

File: src/pages/PostJobs/PostJobs.tsx [hunk 4]
```diff
@@ -17,8 +17,9 @@ const PostJobs = () => {
     <ErrorBoundary>
       <div className="job_page">
         <div className="content">
-          <Product product={ product } isSelected={true} />
-          <Payment product={ product }/>
+          {/*<Product product={ product } isSelected={true} />
```
Comment (maharjunm) [inline src/pages/PostJobs/PostJobs.tsx]: please remove commented code.

**Review by maharjunm — APPROVED** [review]
